Source/Cryptlex.Net/Entities/Event.cs

```diff
@@ -8,15 +8,12 @@ namespace Cryptlex.Net.Entities
     public class Event
     {
         [JsonPropertyName("event")]
```

Comment:
Can't find this entity on Crytplex webAPI. Is it "event" or "type" ?

Reply:
Hmm, not really sure how this class got here... it also has no references. Probably just dead code

Reply:
Nevermind, class is related to Webhooks